### PR TITLE
OSDOCS-176: Added examples of provisioning template applications and …

### PR DESCRIPTION
**Pull request by bergerhoffer**
…service bundles

@openshift/team-documentation For peer review please.

Preview: http://file.rdu.redhat.com/~ahoffer/2019/OSDOCS-176-provision/applications/service_brokers/provisioning-template-application.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@johnkim76 or @shawn-hurley: Can you take a look at these provisioning examples and also configuring the ASB?

Preview: http://file.rdu.redhat.com/~ahoffer/2019/OSDOCS-176-provision/applications/service_brokers/provisioning-template-application.html

**Comment by bergerhoffer** [comment]
Got okay from SME, just waiting on peer review.

File: modules/sb-provision-service-bundle.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+. Create a service binding.
++
+.. From the *Provisioned Services* page, click *dh-postgresql-apb* and click
+*Create Service Binding*.
+.. Review the default service binding name and click *Create*.
++
+This creates a new secret for binding using the name provided.
+
+. Review the secret that was created.
+.. Navigate to *Workloads* -> *Secrets* and verify that a secret was created,
```
Comment (huffmanca) [inline modules/sb-provision-service-bundle.adoc]: I think this and the next line should be reworked. Either breaking it into two sentences, or moving the secret name. It feels clunky as is. What about:

> ...verify that a secret, named **dh-postgresql-apb**, was created.

File: modules/sb-provision-template-application.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+[id='sb-provision-template-application-{context}']
+= Provisioning template applications
+
+The following procedure provisions an example PostgreSQL template application
+that was made available by the template service broker.
+
+.Prerequisites
+
+* You have enabled service catalog.
```
Comment (huffmanca) [inline modules/sb-provision-template-application.adoc]: Same comment here about needing "the" before service catalog.

File: modules/sb-provision-template-application.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+. Create a service binding.
++
+.. From the *Provisioned Services* page, click *postgresql-ephemeral* and click
+*Create Service Binding*.
+.. Review the default service binding name and click *Create*.
++
+This creates a new secret for binding using the name provided.
+
+. Review the secret that was created.
+.. Navigate to *Workloads* -> *Secrets* and verify that a secret was created,
```
Comment (huffmanca) [inline modules/sb-provision-template-application.adoc]: Same comment applies here about this sentence feeling clunky. Possibly move the secret name to immediately after where it's mentioned:

> ...verify that a secret, named **postgresql-ephemeral**, was created.

File: modules/sb-configuring-ansible-service-broker-options.adoc [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+|`brokerNodeSelector`
+|The node selector string used for the broker's deployment.
+|`''`
+
+|`registries`
+|Expressed as a yaml list of broker registry configs, allowing the user to configure the image registries the broker will discover and source its APBs from.
+|Default registries array:
+----
+- type: dockerhub
```
Comment (huffmanca) [inline modules/sb-configuring-ansible-service-broker-options.adoc]: I'm not certain if there's anything that be done about this source block, but it doesn't wrap and renders quite poorly.

**Review by huffmanca — COMMENTED** [review]
Just a few sentence change suggestions. Thanks for submitting this!

**Comment by johnkim76** [comment]
@bergerhoffer 
LGTM

File: modules/sb-configuring-ansible-service-broker-options.adoc (hunk 4 above)
Comment (bergerhoffer) [reply modules/sb-configuring-ansible-service-broker-options.adoc]: I moved it below the table, let me know what you think!

**Pull request merged**